triage_process.md

```diff
@@ -0,0 +1,54 @@
+Assigned pull requests without scope label|[Core/Provider assigned pull requests without scope label](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+archived%3Afalse+-label%3Atest+-label%3Aperformance+-label%3Abug+-label%3Aenhancement+-label%3Adocumentation+-label%3Arefactoring+-label%3Acleanup+-label%3A%22technical+debt%22+-repo%3AManageIQ%2Fmanageiq-release+-repo%3AManageiq%2Fbugzilla_mirror+-repo%3AManageiq%2Fpolisher+-repo%3AManageiq%2Fwrapanapi+-repo%3AManageiq%2Fmanageiq-ui-classic+-repo%3AManageiq%2Fmanageiq-performance+or+-repo%3AManageiq%2Fmanageiq-api+or+-repo%3AManageIQ%2Fui-components+or+-repo%3AManageIQ%2Fmanageiq-consumption+-repo%3AManageIQ%2Freact-ui-components+org%3AManageIQ+-repo%3AManageIQ%2Fmanageiq-v2v+-repo%3AManageiq%2Fmanageiq-ui-service+-repo%3AManageIQ%2Fmanageiq_docs+-repo%3AManageIQ%2Fintegration_tests+-repo%3AManageIQ%2Fmanageiq-v2v-conversion_host+user%3AManageIQ+)|[UI/API assigned pull requests without scope label](https://github.com/issues?page=2&q=is%3Apr+-label%3A%22help+wanted%22+-label%3Arefactoring+-label%3A%22technical+debt%22+-label%3Abug+-label%3Aenhancement+repo%3AManageiq%2Fmanageiq-ui-classic+repo%3AManageiq%2Fmanageiq-api+repo%3AManageIQ%2Fui-components+repo%3AManageIQ%2Freact-ui-components+repo%3AManageiq%2Fmanageiq-ui-service+is%3Aopen&utf8=%E2%9C%93)
+Stale pull requests|[Core/Provider stale pull requests](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+label%3Astale+archived%3Afalse+user%3AManageIQ+)|[UI/API stale pull requests](https://github.com/pulls?utf8=%E2%9C%93&q=is%3Aopen+is%3Apr+label%3Astale+archived%3Afalse+user%3AManageIQ+)
+
+
+
+## Issue Triage
+* Triage unassigned issues
+  * Assign the issue.  There are three possible scenarios.
+    * Assign someone to at least get in on the radar.  This *does not* mean that the assignee is expected to start work on the issue.
+    * Apply the `help wanted` label.  This means that while the issue is valid, the team does not have plans to work on it in the near future.
+    * Do not assign to anyone and revisit the issue.  In other words, we want to be sure that the issue does not get ignored in perpetuity.
+  * Close invalid issues with a comment and/or label as to why it has been closed.
+* Review issues with assignee, but no scope label.
+  * Apply [scope labels](https://www.manageiq.org/docs/guides/labels).
```

Comment:
Instead of a straight URL you can use a relative path to the labels document.  I believe this should translate properly on the website.  So, something like

[scope labels](labels.md)